hazelcast/src/main/java/com/hazelcast/nio/serialization/GenericRecord.java

```diff
@@ -269,13 +296,14 @@
          * @param classDefinition of the portable that we will create
          * @return GenericRecordBuilder for Portable format
          */
-        static Builder portable(ClassDefinition classDefinition) {
+        static Builder portable(@Nonnull ClassDefinition classDefinition) {
```

Comment:
still missing here

Reply:
@cangencer I  updated the pr accordingly